tests/modules/check.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S nix run 'github:clhodapp/nix-runner/7b56158f7ab9fd7806068c6571833210e063df19' --
+#!package nixpkgs#bash
+#!package nixpkgs#home-manager
+#!command bash
+
+self=$(realpath "$(dirname "$0")")
+
+systems=(
+	"x86_64-linux"
+	"aarch64-linux"
+	"x86_64-darwin"
+	"aarch64-darwin"
+)
+
+failures=()
+
+ok=true
+
+echo "=== HOME MANAGER ==="
+for system in "${systems[@]}"; do
+	if ! home-manager --no-write-lock-file build --flake ${self}/home-manager#nixvim-${system}; then
+		failures+=("home-manager-${system}")
+		ok=false
+	fi
+done
+
+if [[ $ok = false ]]; then
+	echo -e "\nFAILURES: ${failures[@]}"
+	exit 1
+fi
```

Comment:
Can't we make a derivation out of this ?

Reply:
I tried, but was not able because home-manager was not able to find the nix profile directory for the user, which seems coherent as I don't think that in derivations a full nix environement exists